### PR TITLE
karma: Add karma test to test-project run

### PR DESCRIPTION
**Pull request by palfrey**

### CODE REVIEW

**Review by thheller — CHANGES_REQUESTED** [review]

File: test-project/ci-run.sh [hunk 1]
```diff
@@ -8,6 +8,11 @@ npx shadow-cljs --cli-info
...
 
-npx shadow-cljs release reagent test-node --verbose
+npx shadow-cljs release reagent test-node test-karma --verbose
 
 node out/test-node/script.js
+
+apt-get update
+apt-get install chromium --no-install-recommends -y
+
+CHROME_BIN=/usr/bin/chromium npx karma start --single-run
```
[the remark this answers is not in the document]
Comment (palfrey) [reply test-project/ci-run.sh:18]: That's going to be a little tricky, given this script runs in `thheller/cljs-dev` and I'm not seeing the source for that anywhere?

[the remark this answers is not in the document]
Comment (thheller) [reply test-project/ci-run.sh:18]: Well happy to take any other image that has all the stuff needed installed. I can't even remember how I created that image. Could probably use an update on that one too.

Just needs java, lein, node, npm.

[the remark this answers is not in the document]
Comment (palfrey) [reply test-project/ci-run.sh]: Turns out CircleCI provide some pretty decent images these days, and one of those fits the bill. Fixed!

[the remark this answers is not in the document]
Comment (palfrey) [reply test-project/ci-run.sh]: Any further thoughts on this @thheller?

**Review by thheller — APPROVED** [review]

**Comment by thheller** [comment]
Sorry for sitting so long on this. I'm really not a fan of karma but I guess it makes sense given that other people use it.

**Pull request merged**